common.sh

```diff
@@ -187,8 +187,18 @@ if [ "${UPSTREAM_IRONIC:-false}" != "false" ] ; then
     export BAREMETAL_OPERATOR_LOCAL_IMAGE=${BAREMETAL_OPERATOR_LOCAL_IMAGE:-"quay.io/metal3-io/baremetal-operator"}
 fi
 
+# Default the build tags used when building from the source the OpenShift installer
+export OPENSHIFT_INSTALLER_BUILD_TAGS=${OPENSHIFT_INSTALLER_BUILD_TAGS:-"libvirt baremetal"}
+
+# Default installer command extracted
+export OPENSHIFT_INSTALLER_CMD=${OPENSHIFT_INSTALLER_CMD:-"openshift-baremetal-install"}
```

Comment:
I wonder if these really need to be common - `build_installer` is only used in `03_build_installer.sh` so I wonder if it would be simpler/clearer if we moved the function into `03_build_installer.sh` and defined the build variables (and defaults) in that script directly?

Reply:
Probably the same for `clone_installer` and `OPENSHIFT_INSTALLER_PR` - not necessarily blocking on that refactor, but it seems like this might be a good time to do it?

Reply:
Yep, I've moved everything but `OPENSHIFT_INSTALLER_CMD`, since `OPENSHIFT_INSTALLER` seems to be more widely used and complicated to move